met-api/src/met_api/models/survey.py

```diff
@@ -45,10 +46,21 @@ def get_open(cls, survey_id) -> Survey:
         return survey
 
     @classmethod
-    def get_surveys_paginated(cls, pagination_options: PaginationOptions, search_text='', unlinked=False):
+    def get_surveys_paginated(  # pylint: disable=too-many-arguments
+        cls, pagination_options: PaginationOptions, search_text='', unlinked=False, is_admin=False, exclude_hidden=False
+    ):
         """Get surveys paginated."""
         query = db.session.query(Survey).join(Engagement, isouter=True).join(EngagementStatus, isouter=True)
 
+        null_value = None
+        # exclude hidden surveys from being fetched by users other than admins
+        if is_admin:
```

Comment:
extract this section into its own function

Reply:
and perhaps go for if 
is_admin or exclude_hidden: 
filter

Reply:
moved this logic to the service instead of the model